### PR TITLE
Remove ex scalers from team_scaleway

### PR DESCRIPTION
**Pull request by QuentinBrosse**
##### SUMMARY
To stop receiving bot notifications as we are not working at Scaleway anymore. :)

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
- cloud/scaleway

### CODE REVIEW

**Comment by QuentinBrosse** [comment]
cc @kindermoumoute @remyleone

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -374,4 +374,4 @@ macros:
   team_netscaler: chiradeep giorgos-nikolopoulos
   team_netvisor: Qalthos amitsi csharpe-pn pdam preetiparasar
   team_networking: NilashishC Qalthos danielmellado ganeshrn justjais trishnaguha
-  team_scaleway: DenBeke QuentinBrosse abarbare jerome-quere kindermoumoute remyleo
+  team_scaleway: abarbare kindermoumoute remyleone
```
Comment (kindermoumoute) [inline .github/BOTMETA.yml:377]: 🙏 

```suggestion
  team_scaleway: abarbare remyleone
```

**Comment by dericcrago** [comment]
thanks @QuentinBrosse, @remyleone, & @kindermoumoute!

**Pull request merged**